app/com/lynxanalytics/biggraph/graph_operations/ExecuteSQL.scala

```diff
@@ -98,13 +113,7 @@ object ExecuteSQL extends OpFromJson {
     val plan = getLogicalPlan(sqlQuery, protoTables)
     val minimizedProtoTables = ProtoTable.minimize(plan, protoTables)
     val tables = minimizedProtoTables.mapValues(protoTable => protoTable.toTable)
-    try {
-      ExecuteSQL.run(sqlQuery, SQLHelper.stripComment(plan.schema), tables)
-    } catch {
-      // The exception is thrown on the plan.schema call
-      case a: UnresolvedException[_] =>
-        throw UnresolvedColumnException(s"${a.treeString} column cannot be found", a)
```

Comment:
UnresolvedException no longer has a tree. https://github.com/apache/spark/commit/32a523b56fc329e858f3a463314c13e3710be5e0 ([SPARK-34234](https://issues.apache.org/jira/browse/SPARK-34234)) I don't know whether we will miss it. Anyway it's gone.